packages/mongo-nav/README.md

```diff
@@ -204,3 +160,71 @@ export interface URLSInterface {
   };
 }
 ```
+
+##### DataInterface
+
+```typescript
+export interface AccountInterface {
+  email: string;
+  firstName: string;
+  lastName: string;
+  openInvitations?: number;
+  username?: string;
+  admin?: boolean;
+}
+
+export interface ProjectInterface {
+  projectId: string;
+  projectName: string;
+  planType?: PlanType;
+  orgId?: string;
+}
+
+export interface CurrentProjectInterface extends ProjectInterface {
+  alertsOpen: number;
+  chartsActivated: boolean;
+  status?: ProjectStatus;
+}
+
+export interface OrganizationInterface {
+  orgId: string;
+  orgName: string;
+  planType: PlanType;
+}
+
+export interface CurrentOrganizationInterface extends OrganizationInterface {
+  paymentStatus?: OrgPaymentLabel;
+}
+
+export interface DataInterface {
+  readonly account: AccountInterface;
```

Comment:
`readonly` is more of a TypeScript-ism and not something we enforce for non-TS users

Reply:
👍